src/main/java/core/basesyntax/figures/Shape.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.figures.interfaces.Drawable;
+import core.basesyntax.figures.interfaces.ShapeCalculable;
+
+public class Shape implements Drawable, ShapeCalculable {
+    protected String color;
+
+    Shape(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("It's just a shape");
+    }
+
+    @Override
+    public double calculateArea() {
+        return 0;
+    }
+}
```

Comment:
If we do not plan to have instance of parent class it's better to make it abstract
And then we may omit methods overriding and delegate it to children classes
```suggestion
public abstract class Shape implements Drawable, ShapeCalculable {
    protected String color;

    Shape(String color) {
        this.color = color;
    }
}
```